<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>spoton_rosetta</class>
 <widget class="QMainWindow" name="spoton_rosetta">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>522</width>
    <height>815</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GoldBug: Rosetta CryptoPad</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../Icons/icons.qrc">
    <normaloff>:/nuvola/goldbug.png</normaloff>:/nuvola/goldbug.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <property name="spacing">
     <number>5</number>
    </property>
    <property name="margin">
     <number>5</number>
    </property>
    <item>
     <widget class="QLabel" name="banderole_rosetta">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>40</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: qlineargradient(spread:reflect, x1:0.023, y1:0.988636, x2:0.028, y2:0.011, stop:0 rgba(218, 232, 255, 255), stop:1 rgba(170, 85, 255, 255));

color: rgb(255, 255, 255);


</string>
      </property>
      <property name="text">
       <string>Rosetta CryptoPad</string>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <property name="indent">
       <number>5</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <property name="spacing">
       <number>3</number>
      </property>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>&amp;Name</string>
        </property>
        <property name="buddy">
         <cstring>name</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="name"/>
      </item>
      <item>
       <widget class="QPushButton" name="save">
        <property name="styleSheet">
         <string notr="true">QPushButton {
 border: 2px solid gray;
 background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                   stop: 0 #E0E0E0, stop: 1 #FFFFFF);
 border-radius: 5px;
 margin: 2px;
 padding: 2px;
 }


QPushButton:hover { 
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                   stop: 0 rgb(254,229,172), stop: 1 #FFFFFF);
background-position: top center; }

QPushButton:pressed {
     background-color: darkgrey;
     border-style: inset;
 } </string>
        </property>
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_7">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>10</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="copy">
        <property name="styleSheet">
         <string notr="true">QPushButton {
 border: 2px solid gray;
 background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                   stop: 0 #E0E0E0, stop: 1 #FFFFFF);
 border-radius: 5px;
 margin: 2px;
 padding: 2px;
 }


QPushButton:hover { 
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                   stop: 0 rgb(254,229,172), stop: 1 #FFFFFF);
background-position: top center; }

QPushButton:pressed {
     background-color: darkgrey;
     border-style: inset;
 } </string>
        </property>
        <property name="text">
         <string>Copy My &amp;Rosetta Keys</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="title">
       <string>Add New Contact</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="spacing">
        <number>5</number>
       </property>
       <property name="margin">
        <number>5</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QTextEdit" name="newContact">
           <property name="styleSheet">
            <string notr="true">background-color: qlineargradient(spread:pad, x1:0.968, y1:0.932, x2:0.513, y2:0.5, stop:0 rgba(183, 235, 255, 255), stop:1 rgba(255, 255, 255, 255)); </string>
           </property>
           <property name="tabChangesFocus">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QPushButton" name="add">
               <property name="styleSheet">
                <string notr="true">QPushButton {
 border: 2px solid gray;
 background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                   stop: 0 #E0E0E0, stop: 1 #FFFFFF);
 border-radius: 5px;
 margin: 2px;
 padding: 2px;
 }


QPushButton:hover { 
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                   stop: 0 rgb(254,229,172), stop: 1 #FFFFFF);
background-position: top center; }

QPushButton:pressed {
     background-color: darkgrey;
     border-style: inset;
 } </string>
               </property>
               <property name="text">
                <string>Add</string>
               </property>
               <property name="icon">
                <iconset resource="../Icons/icons.qrc">
                 <normaloff>:/nuvola/add.png</normaloff>:/nuvola/add.png</iconset>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="clearContact">
               <property name="styleSheet">
                <string notr="true">QPushButton {
 border: 2px solid gray;
 background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                   stop: 0 #E0E0E0, stop: 1 #FFFFFF);
 border-radius: 5px;
 margin: 2px;
 padding: 2px;
 }


QPushButton:hover { 
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                   stop: 0 rgb(254,229,172), stop: 1 #FFFFFF);
background-position: top center; }

QPushButton:pressed {
     background-color: darkgrey;
     border-style: inset;
 } </string>
               </property>
               <property name="text">
                <string>Clear</string>
               </property>
               <property name="icon">
                <iconset resource="../Icons/icons.qrc">
                 <normaloff>:/nuvola/empty-trash.png</normaloff>:/nuvola/empty-trash.png</iconset>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QFormLayout" name="formLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <property name="verticalSpacing">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_6">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>24</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 127);</string>
        </property>
        <property name="text">
         <string>1</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <property name="spacing">
         <number>5</number>
        </property>
        <item>
         <widget class="QRadioButton" name="encrypt">
          <property name="layoutDirection">
           <enum>Qt::RightToLeft</enum>
          </property>
          <property name="text">
           <string>&amp;Encrypt</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="decrypt">
          <property name="text">
           <string>&amp;Decrypt</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_5">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>24</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(170, 170, 255);</string>
        </property>
        <property name="text">
         <string>4</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_7">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>24</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(170, 170, 255);</string>
        </property>
        <property name="text">
         <string>2</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_8">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>24</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 127);</string>
        </property>
        <property name="text">
         <string>3</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>5</number>
        </property>
        <property name="leftMargin">
         <number>10</number>
        </property>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>for/from the &amp;Contact:</string>
          </property>
          <property name="buddy">
           <cstring>contacts</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="contacts">
          <property name="sizeAdjustPolicy">
           <enum>QComboBox::AdjustToContents</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="deleteContact">
          <property name="styleSheet">
           <string notr="true">QPushButton {
 border: 2px solid gray;
 background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                   stop: 0 #E0E0E0, stop: 1 #FFFFFF);
 border-radius: 5px;
 margin: 2px;
 padding: 2px;
 }


QPushButton:hover { 
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                   stop: 0 rgb(254,229,172), stop: 1 #FFFFFF);
background-position: top center; }

QPushButton:pressed {
     background-color: darkgrey;
     border-style: inset;
 } </string>
          </property>
          <property name="text">
           <string>Delete Contact</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="rename">
          <property name="styleSheet">
           <string notr="true">QPushButton {
 border: 2px solid gray;
 background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                   stop: 0 #E0E0E0, stop: 1 #FFFFFF);
 border-radius: 5px;
 margin: 2px;
 padding: 2px;
 }


QPushButton:hover { 
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                   stop: 0 rgb(254,229,172), stop: 1 #FFFFFF);
background-position: top center; }

QPushButton:pressed {
     background-color: darkgrey;
     border-style: inset;
 } </string>
          </property>
          <property name="text">
           <string>Rename</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>10</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="2" column="1">
       <widget class="QSplitter" name="splitter">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="childrenCollapsible">
         <bool>false</bool>
        </property>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Input</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="spacing">
           <number>5</number>
          </property>
          <property name="margin">
           <number>5</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <property name="spacing">
             <number>5</number>
            </property>
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>&amp;Cipher Type:</string>
              </property>
              <property name="buddy">
               <cstring>cipher</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="cipher">
              <property name="sizeAdjustPolicy">
               <enum>QComboBox::AdjustToContents</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>&amp;Hash Type:</string>
              </property>
              <property name="buddy">
               <cstring>hash</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="hash">
              <property name="sizeAdjustPolicy">
               <enum>QComboBox::AdjustToContents</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="sign">
              <property name="text">
               <string>&amp;Sign message.</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="clearInput">
              <property name="styleSheet">
               <string notr="true">QPushButton {
 border: 2px solid gray;
 background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                   stop: 0 #E0E0E0, stop: 1 #FFFFFF);
 border-radius: 5px;
 margin: 2px;
 padding: 2px;
 }


QPushButton:hover { 
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                   stop: 0 rgb(254,229,172), stop: 1 #FFFFFF);
background-position: top center; }

QPushButton:pressed {
     background-color: darkgrey;
     border-style: inset;
 } </string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="../Icons/icons.qrc">
                <normaloff>:/nuvola/empty-trash.png</normaloff>:/nuvola/empty-trash.png</iconset>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QTextEdit" name="input">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
            </property>
            <property name="tabChangesFocus">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Output</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="spacing">
           <number>5</number>
          </property>
          <property name="margin">
           <number>5</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <property name="spacing">
             <number>5</number>
            </property>
            <item>
             <spacer name="horizontalSpacer_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>10</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="clearOutput">
              <property name="styleSheet">
               <string notr="true">QPushButton {
 border: 2px solid gray;
 background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                   stop: 0 #E0E0E0, stop: 1 #FFFFFF);
 border-radius: 5px;
 margin: 2px;
 padding: 2px;
 }


QPushButton:hover { 
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                   stop: 0 rgb(254,229,172), stop: 1 #FFFFFF);
background-position: top center; }

QPushButton:pressed {
     background-color: darkgrey;
     border-style: inset;
 } </string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="../Icons/icons.qrc">
                <normaloff>:/nuvola/empty-trash.png</normaloff>:/nuvola/empty-trash.png</iconset>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QTextBrowser" name="output">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="tabChangesFocus">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="layoutWidget">
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <property name="spacing">
           <number>5</number>
          </property>
          <item>
           <widget class="QPushButton" name="convert">
            <property name="styleSheet">
             <string notr="true">QPushButton {
 border: 2px solid gray;
 background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                   stop: 0 #E0E0E0, stop: 1 #FFFFFF);
 border-radius: 5px;
 margin: 2px;
 padding: 2px;
 }


QPushButton:hover { 
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                   stop: 0 rgb(254,229,172), stop: 1 #FFFFFF);
background-position: top center; }

QPushButton:pressed {
     background-color: darkgrey;
     border-style: inset;
 } </string>
            </property>
            <property name="text">
             <string>Convert</string>
            </property>
            <property name="icon">
             <iconset resource="../Icons/icons.qrc">
              <normaloff>:/nuvola/refresh.png</normaloff>:/nuvola/refresh.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="copyConverted">
        <property name="styleSheet">
         <string notr="true">QPushButton {
 border: 2px solid gray;
 background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                   stop: 0 #E0E0E0, stop: 1 #FFFFFF);
 border-radius: 5px;
 margin: 2px;
 padding: 2px;
 }


QPushButton:hover { 
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                   stop: 0 rgb(254,229,172), stop: 1 #FFFFFF);
background-position: top center; }

QPushButton:pressed {
     background-color: darkgrey;
     border-style: inset;
 } </string>
        </property>
        <property name="text">
         <string>Copy Conversion</string>
        </property>
        <property name="icon">
         <iconset resource="../Icons/icons.qrc">
          <normaloff>:/nuvola/copy.png</normaloff>:/nuvola/copy.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>522</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Close"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="action_Copy"/>
    <addaction name="action_Paste"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_Close">
   <property name="text">
    <string>&amp;Close</string>
   </property>
  </action>
  <action name="action_Empty_Log">
   <property name="text">
    <string>&amp;Empty Log</string>
   </property>
  </action>
  <action name="actionEnable_Log">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>E&amp;nable Log</string>
   </property>
  </action>
  <action name="action_Copy">
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="action_Paste">
   <property name="text">
    <string>Past</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>name</tabstop>
  <tabstop>save</tabstop>
  <tabstop>copy</tabstop>
  <tabstop>newContact</tabstop>
  <tabstop>add</tabstop>
  <tabstop>clearContact</tabstop>
  <tabstop>encrypt</tabstop>
  <tabstop>decrypt</tabstop>
  <tabstop>contacts</tabstop>
  <tabstop>deleteContact</tabstop>
  <tabstop>rename</tabstop>
  <tabstop>cipher</tabstop>
  <tabstop>hash</tabstop>
  <tabstop>sign</tabstop>
  <tabstop>clearInput</tabstop>
  <tabstop>input</tabstop>
  <tabstop>clearOutput</tabstop>
  <tabstop>output</tabstop>
  <tabstop>convert</tabstop>
  <tabstop>copyConverted</tabstop>
 </tabstops>
 <resources>
  <include location="../Icons/icons.qrc"/>
 </resources>
 <connections/>
</ui>
